app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java

```diff
@@ -243,6 +244,7 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 SP.putBoolean(getString(R.string.preference_translucent_status_bar), isChecked);
                 updateTheme();
                 setStatusBarColor();
+                Toast.makeText(SettingsActivity.this, "Please restart the app for better results!", Toast.LENGTH_SHORT).show();
```

Comment:
Please add this string in string.xml

Reply:
okay, so I close this PR, and reopen a new one? RIght?